k8s/images/app/Dockerfile

```diff
@@ -1,18 +1,18 @@
-FROM ubuntu:xenial
+FROM ubuntu:bionic
 
 RUN apt-get update --fix-missing && apt-get -y install curl
 
-# Add the Cloud SDK distribution URI as a package source
-RUN echo "deb http://packages.cloud.google.com/apt cloud-sdk-xenial main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
-
-# Import the Google Cloud Platform public key
-RUN curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+# Set the timezone
+RUN ln -fs /usr/share/zoneinfo/America/Los_Angeles /etc/localtime
 
 # install node
 RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
 
-RUN apt-get update && apt-get -y install nodejs python python-dev python-pip gcc libpq-dev ffmpeg imagemagick ghostscript python-tk make git gettext openjdk-9-jre-headless curl libjpeg-dev wkhtmltopdf google-cloud-sdk fonts-freefont-ttf xfonts-75dpi poppler-utils
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get -y install nodejs python python-dev python-pip gcc libpq-dev ffmpeg imagemagick ghostscript python-tk make git curl libjpeg-dev wkhtmltopdf fonts-freefont-ttf libpng-dev xfonts-75dpi poppler-utils
```

Comment:
I think we need `apt-get update &&` here in front of `apt-get -y install` otherwise it might fail in the future due to some ubuntu repository change, which happened to us before

Reply:
Ah good catch!